### ISSUE TITLE
Add the ability to alter constant learning rate when loading old accelerator states

### ISSUE THREAD

**Issue by AmericanPresidentJimmyCarter** [bghira/SimpleTuner#680]
Here is how I do it in my trainer:

```py
            accelerator.load_state(os.path.join(args.output_dir, path))

            for g in optimizer.param_groups:
                g['lr'] = torch.tensor(args.learning_rate)
            for k, v in lr_scheduler.state_dict().items():
                if k in ('base_lrs', '_last_lr'):
                    v[0] = args.learning_rate
```

In this way, if I change the learning rate in config.env, it will change the learning rate of the optimiser even if I am loading the states.

Not sure how it would work for non-constant.

**Comment by bghira** [comment]
used to just reinitialise the learning rate scheduler - it's commented out in train.py.

basically you just create a new one after accelerator.prepare, so you discard the prev LR states
